### ISSUE TITLE
Add empty alt text for presentational images used on site

### ISSUE THREAD

**Issue by jobara** [IMERSS/xetthecum-storymap#23]
There are some images used on the site, such as the icon array that appears on some pages, should have a `alt=""` so that it isn't exposed to ATs. Other images should also be reviewed for correct alt text usage.

<img width="1047" alt="Screenshot 2024-09-30 at 9 30 27 AM" src="https://github.com/user-attachments/assets/30b1ec49-b40d-4c9e-94c1-e37f845a227f">
